src/com/mendix/widget/GoogleMaps/components/__tests__/Map.spec.ts

```diff
@@ -0,0 +1,201 @@
+import { shallow } from "enzyme";
+import GoogleMap from "google-map-react";
+import { DOM, createElement } from "react";
+
+import { Map, MapProps } from "../Map";
+import { Marker } from "../Marker";
+
+import {
+    EventMock, GeocoderLocationType, GeocoderMock, GeocoderStatus, LatLngBoundsMock,
+    LatLngMock, MapsMock, MarkerMock, MockGoogle
+} from "../../../../../../../tests/mocks/GoogleMaps";
```

Comment:
that is not a good path. Thanks to `"baseUrl": "."` in your tsconfig, you can use paths relative to the root dir of your project.